equinix/services/fabricv4/api/marketplace_subscriptions_api.py

```diff
@@ -265,7 +265,9 @@ def _get_subscription_by_id_serialize(
         _query_params: List[Tuple[str, str]] = []
         _header_params: Dict[str, Optional[str]] = _headers or {}
         _form_params: List[Tuple[str, str]] = []
-        _files: Dict[str, str] = {}
+        _files: Dict[
+            str, Union[str, bytes, List[str], List[bytes], List[Tuple[str, bytes]]]
+        ] = {}
```

Comment:
These changes add support for multiple file parameters; I suspect this isn't relevant to us but it should be backwards-compatible anyway since the `str` type is still a member of the updated union type. https://github.com/OpenAPITools/openapi-generator/pull/19329